main.nf

```diff
@@ -302,16 +270,34 @@ if( params.include_proteins_from_vcf){
         .fromPath( params.vcf )
         .ifEmpty { exit 1, "params.vcf was empty - no input file supplied" }
         .into { input_vcf; input_vcf_neoepitope; input_vcf_neoepitope_II}
-} else {
 
+} else {
     input_vcf = Channel.empty()
     input_vcf_neoepitope = Channel.empty()
     input_vcf_neoepitope_II = Channel.empty()
 }
 
+// Stage config files
+ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
+
+// Has the run name been specified by the user?
+//  this has the bonus effect of catching both -name and --name
+custom_runName = params.name
+if (!(workflow.runName ==~ /[a-z]+_[a-z]+/)) {
+    custom_runName = workflow.runName
+}
+
+if (workflow.profile.contains('awsbatch')) {
```

Comment:
This section looks incomplete relative to the pipeline template.

Reply:
https://github.com/Zethson/MHCquant/commit/ed4d2f780601656b0c440f6b03c63dc024db4c05

Great find!